hey.go

```diff
@@ -190,7 +196,36 @@ func main() {
 			usageAndExit(err.Error())
 		}
 	}
+	// at this point we will read through the CSV and then use that as the url
+
+	csvfile, err := os.Open(fileName)
+	if err != nil {
+		log.Fatalln("couldnt open the csv file", err)
+	}
+	r := csv.NewReader(csvfile)
+	// TODO remove this after i'm done testing
+	break_early := true
+	for {
+		record, err :=  r.Read()
+		nurl := record[0]
+		// ignore any strings that aren't formatted correctly
+		if strings.Contains(nurl, ".com") == false {
```

Comment:
If you're trying to check that it's a valid URL, use `url.Parse`. But you could always just let it fall when it tries to connect.